packages/store/ts/library/render-solidity/tablegen.ts

```diff
@@ -29,7 +29,8 @@ export async function tablegen(config: StoreConfig, outputBaseDirectory: string)
     await formatAndWriteSolidity(output, fullOutputPath, "Generated types file");
   }
 
-  const fullOutputPath = path.join(outputBaseDirectory, `Tables.sol`);
+  // solc expects `/` as path separator, but path.join uses `\` if the user is on Windows
+  const fullOutputPath = path.join(outputBaseDirectory, `Tables.sol`).replace(/\\/g, "/");
```

Comment:
wonder if we can use `path.posix.join` everywhere we need forward slashes:
https://nodejs.org/api/path.html#windows-vs-posix